src/main/java/com/valmiki/adhyapakservice/entity/Courses.java

```diff
@@ -0,0 +1,76 @@
+package com.valmiki.adhyapakservice.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "courses")
+public class Courses {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "course_name")
+    private String courseName;
+
+    @Column(name = "subheading")
+    private String subHeading;
+
+    public Courses() {
+
+    }
+
+    public Courses(int id, String courseName, String subHeading) {
+        this.id = id;
+        this.courseName = courseName;
+        this.subHeading = subHeading;
+    }
+
+    public Courses(String courseName, String subHeading) {
```

Comment:
Same thing. And use builder from lombok rather than creating separate constructors